src/http-metadata.md

```diff
@@ -0,0 +1,80 @@
+---
+---
+
+# Statement Metadata API HTTP Interface
+The table below describes the methods that the Statement Metadata HTTP interface provides.
+
+```
+POST http://www.example.org/api/v2/statementmetadata/:id
+```
+
+To access this interface, you must additionally supply your Basic Auth details with each request in the `Authorization` header. Your Basic Auth details can be found under **Settings** > **Clients**.
+
+The JSON body of these methods is a key-value pair such as the following. Note that when using metadata keys, you need to replace any dots with `&46;` because Mongo does not allow dots in keys. Some examples of where we currently add metadata can be found on [question statements](../guides-assessment-statements) coming from Adapt.
+```
+{
+  "https://learninglocker&46;net/true-false-response": "Yes"
+}
+```
+
+Method | Description
+--- | ---
+[POST /:id](#post-id) | Creates or overwrites a model.
+[PATCH /:id](#patch-id) | Patches a model.
+
+## Routes
+
+### POST /:id
+This route creates or updates the metadata field on a statement that has the specified statement identifier from the URL. A request to this route would look something like the request below.
```

Comment:
We might want a warning here advising people to use patch instead where possible because they might otherwise overwrite metadata set by Learning Locker that is required for certain parts of the UI to work.